library/java/org/chromium/net/FakeX509Util.java

```diff
@@ -0,0 +1,40 @@
+package org.chromium.net;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Fake utility functions for verifying X.509 certificates.
```

Comment:
It would be nice to mention the behavior of this class. For example, if certs A, B, C are passed in to verifyServerCertificate() what will be required for this to return OK vs failure?